concepticondata/conceptlists.tsv

```diff
@@ -440,3 +440,4 @@ Blum-2024-501	Blum, Frederic and Barrientos, Carlos and Zariquiey, Roberto and L
 Smith-2017-679	Smith, Alexander D.	2017		679	basic	English	Burneo languages		Smith2017		This concept list is used to investigate the relatedness and history of the Austronesian languages of Borneo. The study is based primarily on fieldwork in Borneo. It describes the historical phonologies of the languages, evaluates lower- and higher-level linguistic subgroupings, and discusses the history of population movements in Borneo from a linguistic perspective.	i-693
 Zalizniak-2024-4583	Anna Zalizniak and Anna Smirnitskaya and Maksim Russo (Rousseau) and Ilya Gruntov and Timur Maisak and Dmitry Ganenkov and Maria Bulakh and Maria Orlova and Marina Bobrik-Fremke and Oksana Dereza and Tatiana Mikhailova and Maria Bibaeva and Mikhail Voronov	2024		4583	ranked	English	global	http://datsemshift.ru	Zalizniak2024		This is the most recent dump of the DatSemShift database, which was retrieved on February 5 (2024) from the database and then converted to CLDF. The data contains several duplicates, which we have marked by a preceding asterisk, leaving them unlinked to Concepticon and also ignoring them in the network representations.		DatSemShift
 Lastra-1986-381	Lastra de Suárez, Yolanda	1986		381		English	Spanish		Lastra1986		This concept list is the result of a study of the dialectal regions in modern Nahuatl. 	239-248	
+Tjuka-2024-1310	Tjuka, Annika	2024		1310		English	Vietnamese		Tjuka2024		The list includes 1310 concepts for Vietnamese. The concepts are based on the items provided in the [Intercontinental Dictionary Series](https://ids.clld.org/). 		
```

Comment:
Small thing, but maybe the IDS list could be referred to with `:ref:`? (`[Intercontinental Dictionary Series](:ref:Key-2016-1310)` or similar)

Reply:
Sure, I updated it.